packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -400,6 +399,7 @@ class BottomSheet extends Component {
 			children,
 			withHeaderSeparator = false,
 			hasNavigation,
+			onDismiss,
```

Comment:
won't this cause linting warnings because the `onDismiss` variable gets defined but never read?

Reply:
Thanks for asking - it won't since https://github.com/WordPress/gutenberg/pull/41897

We specifically allowed this as an exception for the `no-unused-vars` rule.

Reply:
Cool 👍 Thanks for the context :)